app/views/feedback/_main_feedback_introduction.html.slim

```diff
@@ -0,0 +1,11 @@
+h1= t('feedback.heading')
```

Comment:
Sorry you've misunderstood my point. I meant 

```
# main heading

body

## Tech support heading

tech support body
```

If the choice was to slice it up just to get a paragraph style we could instead make a custom tag